File: core/src/main/java/co/ericp/freewayforecast/weather/Forecast.java
package co.ericp.freewayforecast.weather;

import java.util.List;

import co.ericp.freewayforecast.LatLng;

/**
 * The weather forecast over a given stretch of time in one place.
 */
public class Forecast {
    LatLng coords;
    List<WeatherPoint> points;
    long startTime;
    long endTime;
}

File: core/src/main/java/co/ericp/freewayforecast/weather/Forecast.kt
package co.ericp.freewayforecast.weather

import co.ericp.freewayforecast.LatLon

/**
 * The weather forecast over a given stretch of time in one place.
 */
data class Forecast(
        val coords: LatLon,
        val points: List<WeatherPoint>,
        val startTime: Long,
        val endTime: Long
)
